tidb-lightning/tidb-lightning-configuration.md

```diff
@@ -402,11 +402,11 @@ sql-mode = "ONLY_FULL_GROUP_BY,NO_AUTO_CREATE_USER"
 max-allowed-packet = 67_108_864
 
 # Whether to use TLS for SQL connections. Valid values are:
-#  * ""            - force TLS (same as "cluster") if [tidb.security] section is populated, otherwise same as "false"
-#  * "false"       - disable TLS
-#  * "cluster"     - force TLS and verify the server's certificate with the CA specified in the [tidb.security] section
-#  * "skip-verify" - force TLS but do not verify the server's certificate (insecure!)
-#  * "preferred"   - same as "skip-verify", but if the server does not support TLS, fallback to unencrypted connection
+#  - "": if the [tidb.security] section is populated, TLS is required (same as "cluster"). Otherwise, it uses an unencrypted connection.
```

Comment:
```suggestion
#  - "": if configuration items in the [tidb.security] section are configured, TiDB Lightning requires TLS for SQL connections (same behavior as "cluster"). Otherwise, it uses an unencrypted connection.
```